jrnl/util.py

```diff
@@ -162,6 +162,21 @@ def scope_config(config, journal_name):
     return config
 
 
+def verify_config(config):
+    """
+    Ensures the keys set for colors are valid colorama.Fore attributes.
+    :return: True if all keys are set correctly, False otherwise
+    """
+    all_valid_colors = True
+    for key, color in config["colors"].items():
+        if not getattr(colorama.Fore, color.upper(), None):
+            # TODO: Not sure whether both of these should stay or not.
```

Comment:
This should be addressed before merging.

Reply:
Probably just the print(), but use print(..., file=sys.stderr) to keep it in line with the other jrnl output.